module-2/backend/week_one.md

```diff
@@ -5,30 +5,69 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+<ul>
+  <li>get - Reads info at specified path</li>
+  <li>put - updates or replaces info </li>
+  <li> post - updates or replaces info</li>
+  <li> delete - deletes info</li>
+</ul>
+
 2. What is Sinatra?
+<p>Its a domain specific languang that allows for web application development</p>
+
 4. What is MVC?
+<p>MVC stands for Model View Controller</p>
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+<p>Because its RESTful?</p>
 6. What types of variables are accessible in our view templates without explicitly passing them?
+<p>Instance variables</p>
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
+    @name = "Mr. Ed"

```

Comment:
Right on 7, but for 8 we need to use slightly different syntax if we're going to use a local variable.

```ruby
get '/horses' do
  @count = 1
  erb :index, locals: {name: "Mr. Ed"}
end
```